src/components/game.js

```diff
@@ -36,21 +36,152 @@ class Game extends React.Component {
     const board = [];
     for (let i = 0; i < data.site.siteMetadata.size; i += 1) {
       board[i] = new Array(data.site.siteMetadata.size);
+      for (let j = 0; j < data.site.siteMetadata.size; j += 1) {
+        board[i][j] = 0;
+      }
     }
     this.state = {
       board,
     };
+    this.handleKeyDown = this.handleKeyDown.bind(this);
   }
 
-  newTile = (board) => {
+  componentDidMount() {
+    document.addEventListener('keydown', this.handleKeyDown, false);
+    this.newTile();
+    this.newTile();
+  }
+
+  componentWillUnmount() {
+    document.removeEventListener('keydown', this.handleKeyDown, false);
+  }
+
+  handleKeyDown = (event) => {
+    if (event.key === 'ArrowDown' || event.key === 'ArrowUp'
```

Comment:
Można to zapisać też za pomocą:
```
let arrowKeys = ['ArrowDown', 'ArrowUp', 'ArrowLeft', 'ArrowRight'];
if(arrowKeys .includes(event.key))
```
albo nawet
`if(['ArrowDown', 'ArrowUp', 'ArrowLeft', 'ArrowRight'].includes(event.key))`

Reply:
na razie zostawione jak było